ath.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
+set -euxo pipefail
+cd $(dirname $0)
+
+# https://github.com/jenkinsci/acceptance-test-harness/releases
+export ATH_VERSION=5458.v911b_2f0818ee
```

Comment:
Currently this would be edited manually (as I guess it was in `essentials.yml`). To enable Dependabot, we could make ATH a submodule and somehow find the right container tag; or make a dummy `ath/pom.xml` (unused by actual Maven build) and just have this script `help:evaluate` the dependency version from there.

Reply:
Or we can use updatecli like on the jenkins-infra. Let me craft a draft PR for that. (TL;DR; updatecli is more tedious than dependabot to set up but allows more dependency updates. 3rd alternative is renovabot, never used but it is installed and @timja @NotMyFault know their way with it)

Reply:
updatecli works, Dependabot just has nicer behavior so I prefer it whenever possible.

Reply:
> Or we can use updatecli like on the jenkins-infra

+1

> 3rd alternative is renovabot

renovate supports [wildcard](https://docs.renovatebot.com/configuration-options/#any) matching for non-typical dependency files for custom data source, in this case images from the dockerhub, I'd be in favor of updatecli nevertheless, given we use it widely in different infra repositories 👍🏻